### PR TITLE
Postgresql port

### PR DESCRIPTION
**Pull request by timminata**
Added postgresql as a db provider

### CODE REVIEW

File: GTFS.DB.PostgreSQL/Collections/PostgreSQLAgencyCollection.cs [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+        }
+
+        /// <summary>
+        /// Gets the entity with the given id.
+        /// </summary>
+        /// <param name="entityId"></param>
+        /// <returns></returns>
+        public Agency Get(string entityId)
+        {
+            throw new NotImplementedException();
```
Comment (peetjvv) [inline GTFS.DB.PostgreSQL/Collections/PostgreSQLAgencyCollection.cs:120]: maybe implement this one

File: GTFS.DB.PostgreSQL/Collections/PostgreSQLAgencyCollection.cs [hunk 2]
```diff
@@ -0,0 +1,236 @@
...
+        }
+
+        /// <summary>
+        /// Removes the entity with the given id.
+        /// </summary>
+        /// <param name="entityId"></param>
+        /// <returns></returns>
+        public bool Remove(string entityId)
+        {
+            throw new NotImplementedException();
```
Comment (peetjvv) [inline GTFS.DB.PostgreSQL/Collections/PostgreSQLAgencyCollection.cs]: and this one

File: GTFS.DB.PostgreSQL/Collections/PostgreSQLAgencyCollection.cs (hunk 1 above)
Comment (timminata) [reply GTFS.DB.PostgreSQL/Collections/PostgreSQLAgencyCollection.cs:120]: They have never been implemented though. Not sure we would ever use it

Comment (peetjvv) [reply GTFS.DB.PostgreSQL/Collections/PostgreSQLAgencyCollection.cs:120]: yeah, just realised that they're not necessary for the LINQ statements that we use

File: GTFS/GTFS.csproj [hunk 3]
```diff
@@ -113,6 +110,11 @@
     <Compile Include="StopsToShape\StopAtShapesFinder.cs" />
     <Compile Include="Validation\GTFSFeedValidation.cs" />
   </ItemGroup>
+  <ItemGroup>
+    <Reference Include="System.Data">
+      <HintPath>..\..\..\Program Files (x86)\Reference Assemblies\Microsoft\Framework\.NETFramework\v4.5\System.Data.dll</HintPath>
```
Comment (peetjvv) [inline GTFS/GTFS.csproj:115]: don't think this is such a good idea

**Comment by peetjvv** [comment]
can't find the button, but Approved!

**Pull request merged**